<launch>
   <param name="use_sim_time" value="false"/>
   <node pkg="tf" type="static_transform_publisher" name="rslidar_tf_broadcaster" args="0.181 0.083 0.3 0 0 0 base_footprint rslidar 100" />
   <!--                                                                      args= x       y   z yaw pitch roll-->
   <node pkg="rosbaglive" type="play.py" name="rosbag_play" args="/home/w/njust_driver/bag.bag"/>

    <node name="odom_repair" pkg="njust_pack" type="test_odom.py" output="screen">
      <remap from="odom_input" to="odom"/>
      <remap from="odom_output" to="odom_new"/>
   </node>

  <node pkg="hector_mapping" type="hector_mapping" name="hector_mapping"   output="screen">

   <param name="pub_map_odom_transform" value="true"/>
   <param name="map_frame" value="map" />
   <param name="base_frame" value="base_footprint" />
   <param name="odom_frame" value="odom_combined" />
  <!-- Map size / start point -->

   <param name="map_resolution" value="0.03"/>
   <param name="map_size" value="4000"/>
   <param name="map_start_x" value="0.5"/>
   <param name="map_start_y" value="0.5" />
   <param name="map_multi_res_levels" value="2" />

   

    <!-- Map update parameters -->

   <param name="update_factor_free" value="0.4"/>
   <param name="update_factor_occupied" value="0.9" />   
   <param name="map_update_distance_thresh" value="0.4"/>
   <param name="map_update_angle_thresh" value="0.06" />
   <param name="laser_z_min_value" value = "-1.0" />
   <param name="laser_z_max_value" value = "1.0" />

 

   </node>

   
</launch>
